docs/README.md

```diff
@@ -91,13 +91,13 @@ Once you promoted your workspace, no further actions are needed on your part: yo
 
 #### Managing routes
 
-You can manage if you want to include product, navigation and/or apps routes in your sitemap or not. To do that, check the following step by step.
+You can manage if you want to include product, navigation, apps and/or draft routes in your sitemap or not. To do that, check the following step by step.
```

Comment:
I think these changes in the readme are outdated since the disable is now by term.